content/doc/book/installing/index.adoc

```diff
@@ -613,6 +613,35 @@ ZFS integration with Jenkins should be considered.
 
 include::doc/book/installing/_setup-wizard.adoc[]
 
+=== Offline Jenkins Installation
+
+This section describes how to install Jenkins on a machine
+that does not have an internet connection.
+
+To install Jenkins itself, download the appropriate war file and
+transfer it to your machine.
+
+Plugins are a different matter, due to dependency requirements. Two
+obvious choices are: +
+1) Install Jenkins and add required plugins on a network-attached
+computer. Make an archive of the Jenkins directory structure and
+transfer it to the offline machine. +
+2) Transfer Jenkins and plugin files separately and install on offline
+computer.
+
+If you want to transfer the individual plugins, you'll need to retrieve
+all dependencies as well. There are several dependency retrieval scripts
+on Github. For example:
+
+* https://gist.github.com/micw/e80d739c6099078ce0f3 bash script
+* https://gist.github.com/chuxau/6bc42f0f271704cd4e91 forked from above
+* https://gist.github.com/Lucasus/1a6b8df71425c790361c requires Node
+* https://github.com/samrocketman/jenkins-bootstrap-shared only Java is
```

Comment:
This one we could keep, CC @samrocketman

Reply:
@oleg-nenashev @MarkEWaite @samrocketman 
I am waiting for your inputs

Reply:
@rathneesh https://github.com/jenkins-infra/jenkins.io/pull/3380/files#r432399018